### PR TITLE
add override for mobile devices to show smaller swatches

### PR DESCRIPTION
**Pull request by mkoenig-shopify**
### PR Summary: 
Really small PR to make the swatches a bit smaller on mobile. This will optimize the way they look/feel when selecting variants. 

### Why are these changes introduced?
We got feedback that the swatches were too large on mobile at 4.4rem - so we reduced them to 3.6rem

### What approach did you take?
A media class in CSS

### Visual impact on existing themes
Before:
Screenshot 2024-01-24 at 1 30 53 PM

After: 
Screenshot 2024-01-24 at 1 31 05 PM

### Testing steps/scenarios

- [ ] Step 1 check out these changes on my [test shop pdp](https://content-apparel-example.myshopify.com/products/short-sleeve-t-shirt)

### Demo links

- [Store](https://content-apparel-example.myshopify.com/products/short-sleeve-t-shirt)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [x] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: assets/component-swatch-input.css [hunk 1]
```diff
@@ -10,6 +10,12 @@
   forced-color-adjust: none;
 }
 
+@media screen and (max-width: 750px) {
```
Comment (alisterdev) [inline assets/component-swatch-input.css:13]: You don't need the @media query here, we already override the size for the larger screens here: https://github.com/Shopify/dawn/blob/436e1045db73d1dec121dc4f1151984fab516c86/assets/component-product-variant-picker.css#L153-L156

**Review by alisterdev — COMMENTED** [review]
Since this goes against [a11y guidelines](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html), do we have a longer term plan to address this?

File: assets/component-swatch-input.css [hunk 2]
```diff
@@ -1,15 +1,27 @@
 /* swatch-input lives in its own file for reusability of the swatch in other areas than the product form context */
+
```
Comment (ludoboludo) [inline assets/component-swatch-input.css:2]:
```suggestion
```

File: assets/component-swatch-input.css [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+  --swatch-input--size: 3.6rem;
+  font-size: var(--swatch-input--size);
+  width: 44px;
+  height: 44px;
+  display: flex;
+  justify-content: center;
+  align-items: center
+
+}
+
```
Comment (ludoboludo) [inline assets/component-swatch-input.css:24]: Here are a couple funky things I noticed: 
<details>
<summary>Desktop looks a bit off now</summary>


</details>

<details>
<summary>Here is how it looks when receiving focus</summary>


</details>

Comment (tyleralsbury) [reply assets/component-swatch-input.css:24]: The focus rings are going to be a really tricky part of this.   I don't suggest trying to include this in the next release, personally.

**Comment by tyleralsbury** [comment]
@mkoenig-shopify Is this still something we want to try to pursue?

**Comment by mkoenig-shopify** [comment]
closing in favor of https://github.com/Shopify/dawn/pull/3348#pullrequestreview-1939952853

**Pull request closed without merging**